_docs_integrate/integration-example.md

```diff
@@ -79,12 +77,9 @@ To do so, execute `POST /api/v2/Attributes` with the following payload:
 }
 ```
 
-You can query the Connector's address via the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the address is `did:e:pilot.enmeshed.eu:dids:9364c879fcc41c87c6ecbd`.
-{: .notice--info}
-
```

Comment:
why did you remove this? This is still perfectly valid.

Reply:
The Connector address is no longer required for creating an attribute, so why should it be mentioned in the tutorial? Limit the example to the most necessary to minimize complexity.

Reply:
![image](https://github.com/user-attachments/assets/3d15ce65-c972-402b-ba40-b7e4a8628bd3)

?

Reply:
@britsta is the right side in the screenshot even correct?

Reply:
also - do we really don't need the address in the whole page?

Reply:
before you only needed it when creating the attribute, since the update I need it in your screenshot above, if that's what you want and not a bug (it can no longer be “”)

Reply:
but you need it, so you should re-add the information on how to get to the information..

Reply:
in line 82: the response from creating the attribute gives you the address

Reply:
but how should I get the connection that <your-connector-address> is the owner you copied before... makes no sense for me.

Reply:
why do I now need the owner for the request and can no longer “”, is that intentional? 
And I get the id and the owner when creating the attribute, why can't I use both of them.

Reply:
I get: "error.consumption.requests.invalidRequestItem: 'The provided IdentityAttribute belongs to someone else. You can only share own IdentityAttributes.' at 2024-08-23T14:00:11.876Z",
I think this is a bug. If the bug is fixed the owner can be “” and then the owner of the IdentityInfo is not needed anywhere

Reply:
@britsta wdym?

Reply:
It is intended that an empty string can no longer be specified here as the owner, for several reasons:
- The empty string is often used as a placeholder for other RequestItems if the recipient of the Request is not yet known (i.e. if the Request is sent via a RelationshipTemplate and not a Message). To avoid confusion, the empty string should mainly be used for this purpose and no longer as a way of saving typing work for an already know address (e.g. my address).
- With a ShareAttributeRequestItem, we can share an IdentityAttribute that already exists in the database, in which the address of the `owner` is explicitly specified. Normally, it should be copied from the database and can then be inserted in the `attribute` property of the ShareAttributeRequestItem in exactly this form (with specified `owner` and not empty string).
- In the past, it was possible with some RequestItems, if you wanted to cause a bug, that Attributes could be stored in the database that actually had an empty string as `owner`. This should hopefully be fixed by now, but has led us to become stricter here and only want to use an empty string if we have no choice, i.e. do not know an address.

Reply:
Then the error message I get when validating is incorrect or at least misleading.
But I still see no reason to use: `/api/v2/Account/IdentityInfo`

Reply:
I would actually have expected a different error message as well and also found the problem in the code, will be fixed soon.

Reply:
The PR https://github.com/nmshd/runtime/pull/250 has just been merged so that an error message is now displayed that makes more sense.

When creating an IdentityAttribute, the address of the `owner` can be found in the success response, so that you do not necessarily have to use the use case to query the currently used Identity to get the address. I would probably mention it in a `{: .notice--info}` box anyway, because I think it would be beneficial for the reader.

Reply:
I am very dissatisfied, if it says save the id and the user from the response and in the note directly below it says oh if you forgot to save the user you can also do that with the use-case.

Reply:
true, but if you should have copy pasted this why is there a placeholder?